pipeline-model-api/src/main/resources/ast-schema.json

```diff
@@ -156,6 +156,14 @@
         }
       ]
     },
+    "libraries": {
+      "description": "One or more shared library identifiers to load",
+      "type": "array",
+      "items": {
+        "type": "string"
```

Comment:
By supporting both the current array-of-strings and a new format for `LibraryRetriever` - `anyOf` will let us do that easily enough.